x-pack/plugins/fleet/server/services/agents/status.ts

```diff
@@ -5,6 +5,7 @@
  */
 
 import { SavedObjectsClientContract } from 'src/core/server';
+import pMap from 'p-map';
```

Comment:
I think(?) this increased our bundle size a bit. not a blocker but maybe we can use `reduce` or a simple `for` loop to do the sequential promises instead?

Reply:
The CI metrics show no change in size (well 54 total bytes) 

That seems exceptionally low unless we were already using it, but we could confirm by looking at the bundles. 

Then again, the lib is only 2 or 3 kB anyway https://bundlephobia.com/result?p=p-map@4.0.0 so it shouldn't noticably impact anything

Reply:
It's server side so this has no impact on the bundle size. But it's a good reflex to check when we introduce new dependencies 👍

Reply:
Just FYI: in another server-side module (I think package policies) I used `bluebird.map()` to achieve concurrent async calls. At the time `p-map` was suggested as well, but it was not a direct dependency (one defined explicitly via `package.json`. Perhaps that has changed now.

Reply:
@paul-tavares yes this changed it's now a dependencies in kibana `package.json`